Transcripts/EP_25_Michaela.md

```diff
@@ -13,40 +13,66 @@ came into tech, a little bit (about) my experiences with different companies, an
 
 [00:28:00] Uh, but I thought maybe, you know, well, code reviews could be one thing like I'm telling you my, my perspective on how you can do code reviews more effectively. You can also talk about testing for example, or other engineering topics, so everything that actually interests me a lot, and I have a lot of passion for, and that I hope will bring you also some joy in some interesting perspectives. So, have a good day and talk to you in two weeks. Bye bye. 
 
-I hope you enjoyed another episode after sup engineering unlocked podcast. Don't forget to subscribe. And I talk to you again in two weeks. Bye.
\ No newline at end of file
+I hope you enjoyed another episode after software  engineering unlocked podcast. Don't forget to subscribe. And I talk to you again in two weeks. Bye.
+=======
+[00:24:00] Germany, it's okay. If you have like a it's, I think it's moonlighting in English. So you can and do that. Like, if you're, if you're transparent about it and you're telling your manager, then you can do something outside of your regular work hours. And so this is what I -  But I actually started, especially when I was on parental leave, there was more time to actually do something like that and transitioning away. And so, yeah. And so this is actually how, how I started my training business. So I've training and consulting business in the court review, space. and I'm, yes, I'm giving workshops, I'm working with teams and to help them be productive, very efficient and effective with contributes. And this is exactly, you know, it's playing so much to my strengths and to the vision that I had that I want to do continue working with engineering teams, right? But now it's not one company it's really all over the world and it's not like a large corporation. But I'm working. I work with different other large corporations, but I also work with really small companies or with smaller companies actually with companies of all sizes. And so, uh, for my curiosity of how things work, it's very beneficial. And maybe, yeah, well, I haven't talked with you about so much is why code  reviews, right? I actually probably am not going to talk about a lot about code reviews in this episode. Time flies. I talked a lot about my, my journey. I will do say in another episode. Anyway, um, why code reviews? Well, Um, at Microsoft, we were looking at software engineering practices. So we were not only looking at code reviews, but also contributors. We were looking at how people are testing their system, how they are building and also owning the code review tool, the internal culture rate or code flow. And so we were also looking at, um, how are our engineers actually doing code review. Is that beneficial for them? Right? So what we saw is that teams were spending a lot of time on code reviews. You were spending over six hours and this was back then. Now
```

Comment:
Here you see one problem resulting from the merge conflict. Timestamp 24 comes before 26. Also, the end line (see line 69) is in the middle of the transcript.